src/sync/parallel-page-fetcher.ts

```diff
@@ -0,0 +1,45 @@
+import { TaskPayload } from "~/src/sync/sync.types";
+
+const mergeJiraPayload = (jiraPayload1?: any, jiraPayload2?: any) => {
+	if (!jiraPayload1 && !jiraPayload2) {
+		return undefined;
+	}
+
+	// TODO: add more
+	const pullRequests = [...(jiraPayload1?.pullRequests || []), ...(jiraPayload2?.pullRequests || [])];
+	const builds = [...(jiraPayload1?.builds || []), ...(jiraPayload2?.builds || [])];
+
+	if (pullRequests.length == 0 && builds.length == 0) {
```

Comment:
Think we want to change == to ===. If we only use == we are stating we are okay with any value that is 'loosely' equals to meaning a value of '0'/false would return true in this case. ===, strictly equals, ensures this is only true is we receive the numerical 0. #javascript